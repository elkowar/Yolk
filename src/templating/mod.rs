pub mod document;
pub mod element;
mod parser;

pub(crate) const COMMENT_START: &str = "<yolk> ";
