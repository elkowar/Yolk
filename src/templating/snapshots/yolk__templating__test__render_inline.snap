---
source: src/templating/mod.rs
expression: "Document::parse_string(\"foo /* {< string.upper(YOLK_TEXT) >} */\")?.render(&mut eval_ctx)?"
snapshot_kind: text
---
FOO /* {< string.upper(YOLK_TEXT) >} */
